### ISSUE TITLE
A wrong error caught in initializing process?

### ISSUE THREAD

**Issue by hijiangtao** [NightCatSama/vue-slider-component#34]
Hi, there. I set the slider component with following params, which `interval` is specified to 0.005:

```
{
    width: "auto",
    tooltip: 'hover',
    min: 1,
    max: 4,
    interval: 0.005,
    disabled: false,
    clickable: false,
    tooltipStyle: {
        "backgroundColor": "#000",
        "borderColor": "#000"
    }
}
```

The appearance and operation is normal in the interface, but I found a error info in the console with `[Vue-slider warn]: Prop[interval] is illegal, Please make sure that the interval can be divisible`.

I found the logic to judge the effectiveness of interval is `(this.maximum - this.minimum) % this.interval !== 0`, and I think it may be not exactly right since the precision of JavaScript itself can not strictly support such operation's judgement? Such as some integer is JavaScript would be represented as a `bug` number, 4 -> 4.0000000000000000003.

I think you can replace the division part to something else, such as modified the maximum value slightly according to user's interval setting. Hope the information can help you a little bit. Thanks for your effort in contributing in this component.

**Comment by NightCatSama** [comment]
What is the version you are using now?

This bug may be the same as mentioned in #24 .

**Comment by hijiangtao** [comment]
Sorry about that, the version I used is 2.1.2. I updated it to v2.2.2 and the error log disappeared.

I should search it before open the issue, sorry about the bothering.

**Comment by NightCatSama** [comment]
It doesn't matter.

**Comment by Danilmet** [comment]
https://jsfiddle.net/2xy72dod/2477/

When use keyboard, there was the same bug

**Comment by NightCatSama** [comment]
@Danilmet Thank you for your feedback, which was fixed in version 2.6.2.
